src/nimUtils.ts

```diff
@@ -74,9 +74,17 @@ export function getBinPath(tool: string): string {
         var pathparts = (<string>process.env.PATH).split((<any>path).delimiter);
         _pathesCache[tool] = pathparts.map(dir => path.join(dir, correctBinname(tool))).filter(candidate => fs.existsSync(candidate))[0];
         if (process.platform !== 'win32') {
-            let args = process.platform === 'linux' ? ['-f', _pathesCache[tool]] : [_pathesCache[tool]];
             try {
-                let buff = cp.execFileSync('readlink', args);
+                let buff;
+                if (process.platform === 'darwin') {
+                    // dirty hack because the version of readlink that ships with osx doesn't support the -f flag
+                    // from https://stackoverflow.com/questions/1055671/how-can-i-get-the-behavior-of-gnus-readlink-f-on-a-mac#4031502
+                    buff = cp.execFileSync('python', ['-c', 'import os,sys;print(os.path.realpath(sys.argv[1]))', _pathesCache[tool]]);
```

Comment:
I dont think that use python here is a good idea may be it will enough  construct full path from `_pathesCache[tool] + cp.execFileSync('readlink', [_pathesCache[tool]]) ` in case of relative symlink